src/index.html

```diff
@@ -5,8 +5,378 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a href="#home" class="logo header__logo">
+        <img src="images/logo.png" alt="MOYO logo">
+      </a>
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">
+              SMARTPHONES
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops&#38;computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__card">
+        <div class="card card--container" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="iMac"
+            class="card__image"
+          >
+
+          <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```

Comment:
```suggestion
          <h2 class="card__name">
            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
          </h2>
```
too long for one line